indykite/resource_authorization_policy_test.go

```diff
@@ -479,10 +379,11 @@ func testAuthorizationPolicyResourceDataExists(
 			"description":  Equal(data.ConfigNode.GetDescription().GetValue()),
 			"create_time":  Not(BeEmpty()),
 			"update_time":  Not(BeEmpty()),
-
-			"json_config": MatchJSON(expectedJSON),
+			"json":         MatchJSON(expectedJSON),
+			"status": Equal(data.ConfigNode.GetConfig().(*configpb.ConfigNode_AuthorizationPolicyConfig).
+				AuthorizationPolicyConfig.Status.String()),
 		}
 
-		return convertOmegaMatcherToError(MatchAllKeys(keys), attrs)
+		return convertOmegaMatcherToError(MatchKeys(IgnoreExtras, keys), attrs)
```

Comment:
Why you are ignoring extra keys? Cannot you rather fix keys? Setting something extra in Terraform state can cause unexpected behavior. Keep it as strict as possible here

Reply:
It is a little bit of a pain now because of tags, but I can think of something to keep them strict

Reply:
Check `indykite/resource_email_notification_test.go:537` I have there extraKeys attribute and setting it on fly.

Reply:
Nice I did not know it was a thing. I added the same tags to each request to not have to change much of the code, but I will keep that in mind.

Reply:
Well, now you don't test situation when tags are empty. Not really bad, but fixing code of Terraform provider, which is running on client's computer is much harder. Cannot make a fix and release in couple of minutes

Reply:
I will update that